package.json

```diff
@@ -104,9 +104,6 @@
     "qunit-dom": "1.6.0",
     "sass": "1.32.8"
   },
-  "resolutions": {
-    "focus-trap": "6.2.1"
-  },
```

Comment:
Also remove the `focus-trap` dependency and update the lockfile

Reply:
Done :+1: